samples/.gitignore

```diff
@@ -0,0 +1,60 @@
+# This .gitignore file should be placed at the root of your Unity project directory
```

Comment:
We shouldn't have a .gitignore under samples/ folder, we only put them under individual samples folders.